index.html

```diff
@@ -231,7 +231,7 @@ <h2 class="contact-text">Contact me</h2>
               class="input"
             />
 
-            <label for="email">Email</label>
+            <label class="error" for="email">The email must be in lower case</label>
             <input
               id="email"
               title="Email"
```

Comment:
- [ ] For an invalid email input, the error message should pop up close to the submit button.